.github/workflows/test.yaml

```diff
@@ -298,6 +298,9 @@ jobs:
       - name: cargo test (debug; ring; -Z minimal-versions)
         run: cargo -Z minimal-versions test --no-default-features --features=ring --locked
 
+      - name: carg test (debug; no provider; -Z minimal-versions)
+        run: cargo -Z minimal-versions test --no-default-features --locked
```

Comment:
it seems a little odd to do the -Z minimal-versions thing with both no-default-features *and* default features? I would think if default-features works, then no-default-features also must (since it must have the same or fewer dependencies in play?)

Reply:
Yeah I agree it's odd. I was mainly looking to (ab)use an existing `cargo test` invocation. I'll split it into a separate better described item.

Reply:
I added a `cargo test --no-default-features --locked` step to the main CI job along with a descriptive comment about why existing test steps weren't sufficient to cover this particular use-case where you want no built-in providers because you bring your own 3rd party one.